install/kubernetes/helm/istio/charts/mixer/templates/service.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
     istio: mixer
 spec:
   ports:
-  - name: tcp-plain
+  - name: http2-mixer
     port: 9091
   - name: tcp-mtls
```

Comment:
Shouldn't this also have http2 ? Not sure if any testing is done with secure-mixer...

Reply:
what would this be changed to and are these prefixes documented somewhere?